selfdrive/car/interfaces.py

```diff
@@ -163,12 +162,10 @@ def update(self, c: car.CarControl, can_strings: List[bytes]) -> car.CarState:
     ret.canValid = all(cp.can_valid for cp in self.can_parsers if cp is not None)
     ret.canTimeout = any(cp.bus_timeout for cp in self.can_parsers if cp is not None)
 
-    if ret.vEgoCluster == 0.0 and not self.v_ego_cluster_seen:
+    if ret.vEgoCluster == -1:
       ret.vEgoCluster = ret.vEgo
-    else:
-      self.v_ego_cluster_seen = True
 
-    if ret.cruiseState.speedCluster == 0:
+    if ret.cruiseState.speedCluster == -1:
       ret.cruiseState.speedCluster = ret.cruiseState.speed
```

Comment:
Would be nice if we passed in a CarState to the car interface with default values so we could detect if the car interfaces set certain values or not without modifying cereal.